### DISCUSSION TITLE
Can the mac os hard drive be enlarged to be bigger than 100gb?

### DISCUSSION THREAD

**Discussion by rudra-code-creator** [quickemu-project/quickemu#977] [Quickemu] [answered]
Hi there, I created a MacOS catalina vm a few days ago and then I upgraded it to ventura and also installed xcode, docker and a bunch of other developer tools on it. Due to this my mac os hard drive is now almost full (has only 2gb space left) is there any way to make the hard drive bigger?

Any help is appreciated.

**Comment by lj3954** [comment, marked as answer]
> Hi there, I created a MacOS catalina vm a few days ago and then I upgraded it to ventura and also installed xcode, docker and a bunch of other developer tools on it. Due to this my mac os hard drive is now almost full (has only 2gb space left) is there any way to make the hard drive bigger?
> 
> Any help is appreciated.

You can change the disk size in the config file. For example, add `disk_size="200G"` to change the maximum allocated disk size to 200GB. You will have to change the partition size within the VM as well, if possible, or create another partition.
